File: plugin/plugin-loaj/src/main/java/com/dasoops/dasserver/plugin/loaj/entity/po/ReplyDo.java
package com.dasoops.dasserver.plugin.loaj.entity.po;

import com.baomidou.mybatisplus.annotation.TableName;
import com.dasoops.common.entity.dbo.base.BaseDo;
import lombok.Data;
import lombok.EqualsAndHashCode;

import java.io.Serializable;

/**
 * @Title: ReplyPo
 * @ClassPath com.dasoops.dasserver.plugin.loaj.entity.po.ReplyPo
 * @Author DasoopsNicole@Gmail.com
 * @Date 2022/11/11
 * @Version 1.0.0
 * @Description: 回复订单
 * @see BaseDo
 * @see Serializable
 */
@EqualsAndHashCode(callSuper = true)
@TableName(value ="tb_plugin_loaj_reply")
@Data
public class ReplyDo extends BaseDo implements Serializable {

    /**
     * 触发关键词
     */
    private String keyword;

    /**
     * 回复
     */
    private String reply;

    /**
     * 匹配类型(0:全局匹配;1:前缀匹配;2:后缀匹配;3:包含匹配)
     */
    private Integer matchType;
}

File: plugin/plugin-loaj/src/main/java/com/dasoops/dasserver/plugin/loaj/entity/dbo/ReplyDo.kt
package com.dasoops.dasserver.plugin.loaj.entity.dbo

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableName
import com.dasoops.common.entity.dbo.base.BaseDo

@TableName(value = "tb_plugin_loaj_reply")
class ReplyDo : BaseDo() {

    /**
     * 触发关键词
     */
    var keyword: String? = null

    /**
     * 回复
     */
    var reply: String? = null

    /**
     * 匹配类型(0:全局匹配;1:前缀匹配;2:后缀匹配;3:包含匹配)
     */
    var matchType: Int? = null

    /**
     * 忽略大小写
     */
    var ignoreCase: Int? = null

    /**
     * 忽略全半角
     */
    var ignoreDbc: Int? = null

    /**
     * 是否启用
     */
    @TableField("`enable`")
    var enable: Int? = null
}